src/chain/moonbeam/main.ts

```diff
@@ -6,6 +6,7 @@ export const indexer = new Indexer({
   config: {
     chain: ensureEnvVariable('CHAIN'),
     endpoint: ensureEnvVariable('CHAIN_RPC_ENDPOINT'),
+    gateway: 'https://v2.archive.subsquid.io/network/moonbeam-substrate',
```

Comment:
I think it would be best to have that as a constant somewhere because it's used in a lot of places and would make the transition to v3 way easier in the future - `https://v2.archive.subsquid.io/network`